recipes/pyqode.qt/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyqode.qt" %}
+{% set version = "2.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cfe97d85b210f0d77663c9df47bcb25bbb7cbbe2d9f58c0a9fd94e5e1d7daa2f
+
+build:
+  number: 0
```

Comment:
It looks like this might be a `noarch: python` compatible package:

https://conda-forge.org/docs/meta.html#building-noarch-packages